docs/0.4/concepts/security_considerations.md

```diff
@@ -0,0 +1,164 @@
+# Security Considerations
+
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+Splinter provides several levels of security and customizable permissions.
+
+## Cryptographic signing with public/private keys
+
+Like Sawtooth and other distributed ledgers, Splinter uses cryptographic
+signing with public/private keys. Each user and transaction-submitting entity
+has a public key and an encrypted private key.
+
+All transactions that affect shared state are signed with the encrypted private
+key. This includes circuit creation (proposals and votes) and smart contract
```

Comment:
It is important to clearly differentiate between Admin Service and Scabbard details here. They should maybe even be in different sections. We should also add in examples of the formats of data that is signed.

Reply:
I have questions about the two changes requested in this comment:

- "clearly differentiate between Admin Service and Scabbard details here" 

Is adding service-level detail appropriate in a high-level summary of cryptographic signing in Splinter? If so, could you clarify what differentiating info is needed here?

- "the formats of data that is signed"

Please clarify what this means - or provide an example : )

Reply:
Added info on the admin service (as mentioned above).